capable/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn it_works() {
+        let result = 2 + 2;
+        assert_eq!(result, 4);
+    }
+}
```

Comment:
Should this be empty, don't see much harm leaving a test in here but it might be noise in test output
```suggestion
```

Reply:
I'm fine to leave this here for now, the crate isn't actually hooked into anything yet.